node/src/chain_spec.rs

```diff
@@ -65,10 +65,10 @@ pub fn release_config(endowed_accounts: Vec<MltKeysInfo>) -> Result<ChainSpec, S
 
     Ok(ChainSpec::from_genesis(
         // Name
-        "Release",
+        "Mintlayer_TestNet",
         // ID
-        "rel",
-        ChainType::Development, //TODO: should it be custom? I'm not sure about this part.
+        "mlt_test_net",
+        ChainType::Custom("MLTTestNet".into()), //TODO: I don't think this worked at all, it still goes to local_testnet
```

Comment:
I *think* the chains are differentiated by protocol ID, I opened a PR building on top of this one for that #99

Reply:
@iljakuklic  I already merged it